src/components/dropdown-filter/dropdown-filter.scss

```diff
@@ -22,13 +22,22 @@
   box-sizing: border-box;
   cursor: pointer;
   left: 0;
-  padding: 5px 6px;
+  padding: 6px 6px 5px 6px;
   position: absolute;
   width: 100%;
   z-index: $z-dropdown-list;
 
-  &:hover {
-    background-color: $blue-bright;
-    color: $white;
+  .carbon-link__content {
+    color: $grey-dark-blue;
+  }
+
+  &.carbon-link__anchor {
+    &:hover {
+      background-color: #E6E9E7;
```

Comment:
This color should come from a variable

Reply:
okay, I grabbed it from here
https://github.com/Sage/carbon/blob/master/src/components/dropdown/dropdown.scss#L55
so I am not sure which variable should replace it

Reply:
Yeah i just had a grep in the code base. As it was already there its probably fine to be merged although we should look at a colour name for it or replace it for something of a similar colour

Reply:
yeah, `$grey-dark-blue-15     #E2E3E5` is the closest color in Carbon at the moment, but it is still very different from what is in the current Dropdown component.